test/correctness/simd_op_check.cpp

```diff
@@ -1871,6 +1871,16 @@ struct Test {
         check("vmpa(v*.h,r*.b)", hvx_width/2, 2*i32(i16_1) + 3*i32(i16_2));
         check("v*.w += vmpa(v*.h,r*.b)", hvx_width/2, 2*i32(i16_1) + 3*i32(i16_2) + i32_1);
 
+        check("v*:*.h = vtmpy(v*:*.ub, r*.b)", hvx_width/1, 2*i16(in_u8(x + hvx_width - 1)) + 3*i16(in_u8(x + hvx_width)) + i16(in_u8(x + hvx_width + 1)));
```

Comment:
I was changing the bounds for x in https://github.com/halide/Halide/blob/master/test/correctness/simd_op_check.cpp#L195 as

        f.bound(x, PADD, W-PADD).vectorize(x, vector_width);
        f.compute_root();

        // Include a scalar version
        Func f_scalar("scalar_" + name);
        f_scalar(x, y) = e;
        f_scalar.bound(x, PADD, W-PADD);
        f_scalar.compute_root();

        // The output to the pipeline is the maximum absolute difference as a double.
        RDom r(PADD, W-PADD, 0, H);
        Func error("error_" + name);
        error() = cast<double>(maximum(absd(f(r.x, r.y), f_scalar(r.x, r.y))));